src/styles/main.css

```diff
@@ -123,3 +126,11 @@ footer {
 .chooser {
   margin-top: 0.5em;
 }
+
+.preview {
+  background-color: red;
```

Comment:
Can you please update the color? Somebody will be blind. Update the font color to white for example.

<img width="1280" alt="screen shot 2017-12-28 at 2 40 25 pm" src="https://user-images.githubusercontent.com/8147926/34403812-1c1a855a-ebdd-11e7-966f-274ed9b58daa.png">

Reply:
A darker shade of red? Or were you thinking another color entirely?

Reply:
Also, `crimson` or `orangered` is better than `red`

Reply:
Use `crimson` as a background color and `white` as a fore color would be good I think.

Reply:
@andrewda He meant the font color.
My comment is for the background.